worksheet_A/TerminalHacking.cpp

```diff
@@ -40,7 +63,35 @@ int main()
 	}
 
 	// TODO: implement the rest of the game
+	HackingIntro();
+
+	while (PlayerLives > 0) 
+	{
+		cout << "Please Enter Password\n";
+		//getline(cin, Guess);
+		cin >> Guess;
+		Likeness = GetLikeness(Guess, secret);
+		if (Likeness == wordLength)
+		{
+			cout << "Correct Password" << endl;
+			return 0;
+		}
+		
+		PlayerLives -= 1;
+		cout << Likeness << "Letters Match" << endl;
+		cout << "You Have " << PlayerLives << "lives remaining" << endl;
+		cout << endl;
+	}
 
     return 0;
 }
 
+void HackingIntro()
+{
+	//Create Constant Word Length
```

Comment:
The comment doesn't appear to correspond to the code here. In general, your code would benefit from more comments.